target/linux/ramips/dts/GB-PC1.dts

```diff
@@ -0,0 +1,157 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "GB-PC1";
```

Comment:
Checked more thoroughly and found two examples : 
The "mt7621-eval-board" ... and your recent commit for the Phicomm K2P by Chuanhong Guo
Would you like me to add compatible string to the other MT7621 devices?